### ISSUE TITLE
Pretrained Weights For Large VQA Models

### ISSUE THREAD

**Issue by yakobyd** [facebookresearch/mmf#552]
Hello! I am currently using this repository for evaluation of several VQA models, thank you for sharing this code.

Some large models such as MoViE+MCAN, ViLBERT and VisualBERT require heavy computational resources to train (e.g. 8 TitanX GPUs, as reported in the ViLBERT paper). I tried to retrain these models on very small batch sizes, but (as expected) the results were very far from the ones reported in the papers.

I am specifically interested in weights for models that are only trained on the 'train' set of VQA (I am aware that MoViE+MCAN weights are available for training on the 'train_val' set, thanks for that). It seems that currently there are no such weights available and I believe that uploading these will help many researchers (especially the ones who do not posses massive computational resources).

Do you plan to train the above models and add their weights in the future? Or perhaps you even have some of those weights currently available and you are planning to upload them soon, it would help a lot, thanks.

**Comment by apsdehal** [comment]
Hi there,

To reproduce VisualBERT you can try 88k iterations, 56 batch size, lr = 5e-5, with warmup of 2000 steps with warmup_cosine scheduler. That should fit in about 2 GPUs with 16GB memory each. 

For movie+mcan, we have the train+val model in the zoo: https://github.com/facebookresearch/mmf/blob/master/mmf/configs/zoo/models.yaml#L446

Tagging @vedanuj, he will let you know if we have any plans of adding train only model.

**Comment by vedanuj** [comment]
@yakobyd We do not have any immediate plans to train a new model. In order to use large batch sizes with limited GPUs you can try using the `update_frequency` config to accumulate gradients and have a larger effective batch size. More details [here](https://github.com/facebookresearch/mmf/blob/a67ca624be60f15cee75842fb43de001729244ae/mmf/configs/defaults.yaml#L48). Let us know if this helps.
